pkg/liqo-controller-manager/namespace-controller/check_LiqoLabel_presence.go

```diff
@@ -0,0 +1,46 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package namespacectrl
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+)
+
+// Checks if the Namespace which triggers an Event, contains liqoLabel.
+func liqoLabelPresence(labels map[string]string) bool {
```

Comment:
```suggestion
func isLiqoEnabledLabelPresent(labels map[string]string) bool {
```

IMHO this is more clear, but if you don't like it just discard the comment

Reply:
it sounds more clear, thank you